### PR TITLE
Hours logs

### PR DESCRIPTION
**Pull request by KaustubhKhulbe**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -19,11 +19,14 @@
     "@skyra/editable-commands": "^2.1.0",
     "discord.js": "^13.3.1",
     "dotenv-cra": "^3.0.2",
+    "google-spreadsheet": "^3.2.0",
+    "googleapis": "^91.0.0",
```
Comment (nmathira) [inline package.json:23]: idk if I'm stupid but do you really need both dependencies?

File: src/lib/types/augements.d.ts [hunk 2]
```diff
@@ -0,0 +1,20 @@
+import type { GoogleSpreadsheet } from 'google-spreadsheet';
+import type { sheets_v4 } from 'googleapis';
+import type { AwsClient, BaseExternalAccountClient, Compute, GoogleAuth, JWT, UserRefreshClient } from 'googleapis-common';
+import type { Impersonated } from 'googleapis-common/node_modules/google-auth-library';
+
+// declare module "@sapphire/framework" {
```
Comment (nmathira) [inline src/lib/types/augements.d.ts:6]: delete?

File: src/listeners/ready.ts [hunk 3]
```diff
@@ -14,11 +15,30 @@ export class UserEvent extends Listener {
 		});
 	}
 
-	public run() {
+	public async run() {
+                this.container.logger.debug(await this.connectToSpreadsheet() == "200" ? "Connected to GoogleSheets" : "Connection to GoogleSheets Failed, you probably fucked something up")
```
Comment (nmathira) [inline src/listeners/ready.ts:19]: nice one

Comment (nmathira) [reply src/listeners/ready.ts:19]: looks pretty good (partly cuz I wrote half of it

File: src/lib/google-sheets/hours-backend.ts [hunk 4]
```diff
@@ -0,0 +1,63 @@
...
+					} catch (error) {
+						console.log(error);
+					}
+				}
+			} else {
+				send(message, 'Successfully added: ' + user.username);
+			}
+		}
+	);
+}
```
Comment (superstarcoder) [inline src/lib/google-sheets/hours-backend.ts:63]: why are tabs 8 spaces long pls fix later

File: src/listeners/ready.ts (hunk 3 above)
Comment (superstarcoder) [reply src/listeners/ready.ts:19]: > 

🙆🏽‍♂️

**Review by superstarcoder — APPROVED** [review]
looks good mostly

**Pull request merged**